### DISCUSSION TITLE
Propagating recent improvements to Home Assistant OS VM script to past deployment

### DISCUSSION THREAD

**Discussion by dimatx** [tteck/Proxmox#378] [General]
@tteck, based on your change log notes RE: Home Assistant OS VM from [2022-07-24](https://github.com/tteck/Proxmox/blob/main/CHANGELOG.MD), can a HA VM created with a prior version of the script take advantage of what sounds like good improvements?

**Comment by tteck** [comment]
Yep,
Click on VM > Hardware > Hard Disk > Edit
enable the following
![Screenshot 2022-07-24 1 51 55 PM](https://user-images.githubusercontent.com/21342670/180783819-26303359-0202-402a-9cea-c081736cce77.png)
Then reboot VM. I don’t know if previous unused blocks will be removed (doubtful).

**Reply by dimatx** [reply]
Thanks. Will give that a shot!

And thanks for this scripts repository. It is very valuable.

**Comment by xijio** [comment]
+1 on thanks, might be good to add a link to this post in the change log

**Comment by tteck** [comment]
**2022-07-26**

**Changed
Home Assistant OS VM**

Set the real time clock (RTC) to local time.
Disable the USB tablet device (save resources / not needed).

![Screenshot 2022-07-26 5 31 28 PM](https://user-images.githubusercontent.com/21342670/181116111-9a07e169-0602-46f1-ab13-9f90b2967c4a.png)

**Reply by smarthomelawyer** [reply]
Stumbled upon this and made those changes myself. Thanks! I also compared the rest of the configs in the Options with those of mine and I saw a difference in the OS Type. You have Linux 5. x - 2.6 Kernel here and I have Other. My VM has been running without issue on Other for months now, but was curious whether there is a benefit - or  the reasons thereof - for specifying the OS as Linux?

**Reply by tteck** [reply]
This is used to enable special optimization/features for specific operating systems

**Comment by tteck** [comment]
### Changed

- **Home Assistant OS VM** 
  - Validate Storage

![Screenshot 2022-08-12 1 42 54 PM](https://user-images.githubusercontent.com/21342670/184414294-8820b4ba-90d1-4f0f-b17e-093bd94bf7e6.png)

**Comment by tteck** [comment]
### Changed

- **Home Assistant OS VM** 
  - Option to create VM using Stable, Beta or Dev Image

![Screenshot 2022-08-16 11 49 20 AM](https://user-images.githubusercontent.com/21342670/184923768-281decd9-d682-4879-955b-3c09778cbb81.png)
